spring-session-core/spring-session-core.gradle

```diff
@@ -4,6 +4,7 @@ description = "Spring Session"
 
 dependencies {
 	api "org.springframework:spring-jcl"
+	implementation "org.springframework:spring-core"
```

Comment:
Historically, Spring Session has avoided the required dependency on `spring-core` so that the Servlet support can be used even in applications that are not Spring-based. See [`spring-session-sample-misc-hazelcast`](https://github.com/spring-projects/spring-session/tree/main/spring-session-samples/spring-session-sample-misc-hazelcast) as an example of such arrangement.